daemon/stats.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef STATS_H
+#define STATS_H
+
+#include <string>
+
+namespace monitord {
+namespace server {
+
+class cpu_monitor
```

Comment:
Better name would be e.g. `system_resource_monitor`, while it's observing both cpu and memory statistics.